AUTHORS

```diff
@@ -0,0 +1,13 @@
+# This is the list of Homeday-blocks authors
+#
+# This does not necessarily list everyone who has contributed code, to see the full list
+# of contributors, see the revision history in source control.
+# List is order alphabetically based on last name
+
+Ilyas Amezouar <ilyas.amezouar@homeday.de>
+Juan Denis <juan.denis@homeday.de>
+Sinisa Grubor <sinisa.grubor@homeday.de>
+Oleg Ivaniv
+Vinicius Kiatkoski Neves <vinicius.neves@homeday.de>
+Daniele Rapisarda <daniele.rapisarda@homeday.de>
+Ante Scepic <ante.sepic@homeday.de>
```

Comment:
```suggestion
Ante Sepic <ante.sepic@homeday.de>
```

A small typo :)

Reply:
Oh, sorry about that. :see_no_evil: It happens often when I type. Fixed in #201 .